.travis.yml

```diff
@@ -4,10 +4,10 @@ language: c
 env:
```

Comment:
commit message: s/amd/and/
Also: The travis build failed complaining about a syntax error.